src/rpm/package.rs

```diff
@@ -626,10 +626,18 @@ impl RPMPackageMetadata {
 
     #[inline]
     pub fn get_payload_compressor(&self) -> Result<CompressionType, RPMError> {
```

Comment:
Really inconvenient to get an error instead of `CompressionType::None` on uncompressed packages.

Reply:
It depends how the absence of the value is interpreted. It'd be fine if the error was only used in case of unknown vales rather than absence which (afaiu) is perfectly valid.

Reply:
If you're ok with it then :+1:

Can you approve?

Reply:
I'm surprised this is a bug fix inside of the 0.11.0 branch release; normally I'd expect a bug fix to be in its own PR and branch.

Reply:
I know, it's not the best thing.  It's just a small thing I noticed that I wanted to fix before cutting the release

Reply:
Ok... one more change :P

Reply:
yes please